### ISSUE TITLE
Can this Project run under Windows 10 

### ISSUE THREAD

**Issue by Dahandla** [FORTH-ModelBasedTracker/MocapNET#10]

**Comment by AmmarkoV** [comment]
No, this codebase only runs on Linux, the development platform is Ubuntu 18.04

**Comment by simon-eda** [comment]
i run FORTH MocapNET using Windows Subsystem for Linux: 
https://www.microsoft.com/de-at/p/ubuntu-1804-lts/9n9tngvndl3q?rtc=1

you need to install all basics (git clone, pip, anaconda) after installation.

**Comment by AmmarkoV** [comment]
So since it appears that the project can be run using the Windows subsystem for Linux I am closing this issue!

**Comment by JustinB4** [comment]
Don't forget to use the --novisualization argument.

**Comment by chozabu** [comment]
Hmm, Trying to run it on windows10 with WSL2 (ubuntu 20.04)
When I run this command `./MocapNET2LiveWebcamDemo /mnt/d/vidtest.mp4 --novisualization`
I get this output
```
chozabu@chozabu-bigtop:~/ann/MocapNET$ ./MocapNET2LiveWebcamDemo /mnt/d/vidtest.mp4 --novisualization
CPU :  Intel(R) Core(TM) i9-10980XE CPU @ 3.00GHz
GPU :
Visualization disabled
Trying to open source (terminate called after throwing an instance of 'std::logic_error'
  what():  basic_string::_M_construct null not valid
Aborted
chozabu@chozabu-bigtop:~/ann/MocapNET$ Your version is up to date
````
Though everything seems to work fine on my linux machine

**Comment by aanexplus** [comment]
> Hmm, Trying to run it on windows10 with WSL2 (ubuntu 20.04) When I run this command `./MocapNET2LiveWebcamDemo /mnt/d/vidtest.mp4 --novisualization` I get this output
> 
> ```
> chozabu@chozabu-bigtop:~/ann/MocapNET$ ./MocapNET2LiveWebcamDemo /mnt/d/vidtest.mp4 --novisualization
> CPU :  Intel(R) Core(TM) i9-10980XE CPU @ 3.00GHz
> GPU :
> Visualization disabled
> Trying to open source (terminate called after throwing an instance of 'std::logic_error'
>   what():  basic_string::_M_construct null not valid
> Aborted
> chozabu@chozabu-bigtop:~/ann/MocapNET$ Your version is up to date
> ```
> 
> Though everything seems to work fine on my linux machine

Did you fix this error? If not, try using ubuntu 18.04. It worked just fine on my 18.04. I used the CPU version of tensorflow so you might want to try that too.
